app/ledger.js

```diff
@@ -143,15 +143,14 @@ const doAction = (action) => {
 
         case settings.MINIMUM_VISIT_TIME:
           if (action.value <= 0) break
-
-          synopsis.options.minDuration = action.value * 1000
+          ledgerInfo.minDuration = action.value
           updatePublisherInfo()
           break
 
         case settings.MINIMUM_VISITS:
           if (action.value <= 0) break
 
-          synopsis.options.minPublisherVisits = action.value
```

Comment:
@mrose17 ok so the problem right now in master is that it's first stored in ledgerData and then here it's added to ledgerData.synopsisOptions.

If it needs to be in synopsisOptions is the fix moving the initial value from the root of ledgerData?